.github/workflows/coverage.yml

```diff
@@ -3,8 +3,7 @@ on: ['push', 'pull_request']
 name: Test Coveralls
```

Comment:
We would probably like to run this action only on push. Otherwise when opening a pull request it will trigger this action twice.